apps/src/applab/designElements/screen.jsx

```diff
@@ -44,6 +44,9 @@ module.exports = {
     element.style.width = Applab.appWidth + 'px';
     element.style.left = '0px';
     element.style.top = '0px';
+    // We want our screen to be behind canvases
+    element.style.position = 'absolute';
+    element.style.zIndex = -2;
```

Comment:
add comment: "setting any z-index here makes the screen the root of a new stacking context, forcing the screen to appear behind all of its children, regardless of their relative z-index values: http://philipwalton.com/articles/what-no-one-told-you-about-z-index/"

It might also make more sense to just set z-index of zero on this line, to avoid the illusion that the value of -2 matters.